app-editors/vscode/vscode-1.93.1-r1.ebuild

```diff
@@ -68,6 +68,7 @@ RDEPEND="
 	x11-libs/libXScrnSaver
 	x11-libs/pango
 	x11-misc/xdg-utils
+	liveshare? ( dev-libs/icu )
```

Comment:
If the only thing that this does is add a dependency, why isn't it an optfeature instead?

More particularly:
- the discussion on the bugtracker seems to indicate it requires more than just ICU?
- dev-libs/icu is NOT the same as icu69. :) Gentoo provides `dev-libs/icu-74.2` but icu69 would be `=dev-libs/icu-69*` and this was removed from Gentoo in February 2022: f9e554b249cde1ae33d6e09a9fb0b22bcc9aa9c0

Reply:
> why isn't it an optfeature instead?
Wasn't really aware of that feature, I'll change it to that.

>dev-libs/icu is NOT the same as icu69. :)
agreed. But as you can see in my PR message, liveshare sucessfully worked on my machine, and when I did `lsof -P -T -p 9273 | grep -i icu` (`9273` is a PID of one vscode process), it showed that it was using my system installed latest ICU. So the information that it need version `69` is outdated, or the `lsof` command is not doing what I think it does.